global/cta-analytics@1.0.js

```diff
@@ -0,0 +1,41 @@
+!function () {
+  window.$loaded(function (window, document, $, undefined) {
+    var ready = function () {
+      (function trackAnalytics() {
+        function trackElement () {
+          var link = $(this);
+          var name = link.attr('data-name');
+          var category = link.attr('data-category');
+          var action = link.attr('data-action');
+          var source = link.attr('data-source');
+          var label = link.attr('data-label');
+          var source_detail = link.attr('data-source-detail');
+          var href = link.attr('href');
+
+          var properties = { category, action, source, label, source_detail };
+
+          link.one('click', function (e) {
+            e.preventDefault();
+            e.stopImmediatePropagation();
+
+            analytics.track(
+              name,
+              properties,
+              { integrations: { Intercom: false } },
+              function () {
+                return window.location.href = href;
+              }
+            );
+          });
```

Comment:
I think we can implement some of the changes from the new navigation script here. For example:
`link.on('click'` instead of `link.one`
I think we can also remove
```
e.preventDefault();
 e.stopImmediatePropagation();
```

Reply:
This was basically to prevent double submissions for tracking

Reply:
Good catch @dominicaw. 

@chama-ram We should instead just set the button to disabled on click, then re-enable after submission (fail or success).

Reply:
Keen to see this change before approval

Reply:
Agreed on the above. There may be oddities that come in from only binding it on a single event.